solr/test-framework/src/java/org/apache/solr/cloud/MiniSolrCloudCluster.java

```diff
@@ -1299,5 +1301,15 @@ public Builder withTraceIdGenerationDisabled() {
       this.disableTraceIdGeneration = true;
       return this;
     }
+
+    /**
+     * It only makes sense to call this if we are using the alwaysOn tracer. this will randomize the
+     * Span::isRecording check for the alwaysOn tracer so we have coverage of all methods that deal
+     * with span creation without having to enable otel module
+     */
+    private void injectRandomRecordingFlag() {
+      boolean isRecording = LuceneTestCase.random().nextBoolean();
```

Comment:
I believe there is a "rarely()" method or something like that.  I think it should be rare that we add this and not merely 50%.

Reply:
good suggestion. will move to rarely()